src/components/Dropdown/PrioritySelector/PrioritySelector.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import Close from '@material-design-icons/svg/round/close.svg';
+import KeyboardDoubleArrowUp from '@material-design-icons/svg/round/keyboard_double_arrow_up.svg';
+import KeyboardArrowUp from '@material-design-icons/svg/round/keyboard_arrow_up.svg';
+import KeyboardArrowDown from '@material-design-icons/svg/round/keyboard_arrow_down.svg';
+import { bem } from '../../../utils/bem';
+import { Dropdown } from '../Dropdown';
+import { ListItem } from '../../List';
+import { Text } from '../../Text';
+import styles from './PrioritySelector.scss';
+import { IconButton } from '../../Buttons';
+
+const iconMap = {
+    mandatory: KeyboardDoubleArrowUp,
+    important: KeyboardArrowUp,
+    optional: KeyboardArrowDown,
+    exclude: Close,
+};
+
+export type Priority = 'mandatory' | 'important' | 'optional' | 'exclude';
+
+export type PriorityItem = {
+    priority: Priority;
+    label: string;
+    value?: string;
+};
+
+const defaultList: PriorityItem[] = [
+    { priority: 'mandatory', label: 'Mandatory' },
+    { priority: 'important', label: 'Important' },
+    { priority: 'optional', label: 'Optional' },
+    { priority: 'exclude', label: 'Exclude' },
+];
+
+export interface Props<>extends Omit<React.HTMLAttributes<HTMLDivElement>, 'onChange'> {
+    /** Currently selected priority item that indicates the importance of the component. */
+    selectedItem: PriorityItem;
+    /** Array of availible priority items. */
+    list: PriorityItem[];
+    /** Callback function triggered when a new priority is selected. */
+    onChange: (newPriorityItem: PriorityItem) => void;
+    /** Boolean indicating whether the whole badge should be disabled. */
+    isDisabled?: boolean;
+    /** Priority button label name for ARIA labelling */
+    buttonLabel?: string;
+}
+
+const { block, elem } = bem('PrioritySelector', styles);
+export function PrioritySelector({
+    selectedItem,
+    isDisabled = false,
+    onChange,
+    buttonLabel,
+    list = defaultList,
```

Comment:
I don't think you should have this. It introduces language dependent elements, which OneUI shouldn't do. You can use it in the story, or tests, but not in the actual implementation.